pypi_on_github_indexer/__main__.py

```diff
@@ -30,7 +30,10 @@ def parse_args():
                 ("target-dir", "Path in the index repository that is the PyPi root. We are "
                                "assuming GitHub Pages by default.", "docs"),
                 ("do-not-push", "Do not push to the index repo. Set this to whatever value "
-                                "to activate this option.", ""))
+                                "to activate this option.", ""),
+                ("repo-commit", "Hash of specific commit of repo to be installed", None),
+                ("repo-subdirectory", "Subdirectory of repo to be installed as a Python package",
+                 None))
```

Comment:
the style of the rest of the optional args is to leave an empty string as non-set default